src/Illuminate/View/Compilers/ComponentTagCompiler.php

```diff
@@ -341,11 +369,14 @@ public function formatClassName(string $component)
      * Guess the view name for the given component.
      *
      * @param  string  $name
+     * @param  string  $prefix
      * @return string
      */
-    public function guessViewName($name)
+    public function guessViewName($name, $prefix = 'components.')
```

Comment:
Wouldn't this be a breaking change? 🙂

Reply:
Thanks! I think that theoretically it would be a breaking change in the sense of people who override this class. However, when I do a search for ComponentTagCompiler, I see that this class is always instantiated like `new ComponentTagCompiler(/** */)`, so as far as I know people are technically not able to override this. So in that sense it wouldn't be a breaking change.